README.md

```diff
@@ -69,7 +70,12 @@ import Comment from "../Comment"
 
 class Products extends Component {
   render () {
+    let comments = this.props.listings.map( (name, price, index) => (
```

Comment:
`map` receives one object for each item in the array, as the first callback argument. And then we'd need to reference `price` and `name` as properties of this :
```js
let comments = this.props.listings.map( (listItem, index) => (
  <Comment name={listItem.name} price={listItem.price} key={index} />
))
```